clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -80,15 +83,45 @@ class GravitinoVirtualFileSystem(fsspec.AbstractFileSystem):
 
     def __init__(
         self,
-        server_uri=None,
-        metalake_name=None,
-        cache_size=20,
-        cache_expired_time=3600,
+        server_uri: str = None,
+        metalake_name: str = None,
+        options: Dict = None,
         **kwargs,
     ):
+        """Initialize the GravitinoVirtualFileSystem.
+        :param server_uri: Gravitino server URI
+        :param metalake_name: Gravitino metalake name
+        :param options: Options for the GravitinoVirtualFileSystem
+        :param kwargs: Extra args for super filesystem
+        """
         self._metalake = metalake_name
-        self._client = GravitinoClient(
-            uri=server_uri, metalake_name=metalake_name, check_version=False
+        auth_type = (
+            GVFSConfig.DEFAULT_AUTH_TYPE
+            if options is None
+            else options.get(GVFSConfig.AUTH_TYPE, GVFSConfig.DEFAULT_AUTH_TYPE)
+        )
+        if auth_type == GVFSConfig.DEFAULT_AUTH_TYPE:
+            self._client = GravitinoClient(
+                uri=server_uri,
+                metalake_name=metalake_name,
+                check_version=False,
```

Comment:
Why is it False here? I think we should set it to True.

Reply:
Yeah, I will remove this change, it's just for testing.